packages/shared/ReactFeatureFlags.js

```diff
@@ -101,6 +101,8 @@ export const warnAboutSpreadingKeyToJSX = false;
 
 export const warnOnSubscriptionInsideStartTransition = false;
 
+export const enableFizzSuspenseAvoidThisFallback = false;
```

Comment:
You can drop the Fizz part of the flag name if you plan on using the same on for the client (Fiber) anyway.

Reply:
It might be useful to compare the Fizz native implementation vs just on the client ? Or I can change it in a follow-up diff if we do gate the client side too? Or did you want me to gate the client side as well in this PR?

Reply:
The flag will mostly be to disable it for OSS rather than useful for test.

We don't actually use Fizz yet so there's nothing to test other than all of Fizz itself. Also, not sure it makes sense to test a feature as either working or not working.

You can do the client-side in a follow up, I just figured you can avoid a rebase/change in the follow up by naming it now.